test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -689,24 +689,38 @@ public void DescriptorResolver_DoesNotReturnInvalidTagHelpersWhenSpecified()
             Assert.Empty(descriptors);
         }
 
+        public static TheoryData<string> DescriptorResolver_IgnoresSpacesData
```

Comment:
Definitely an interesting approach; turns out its definitely needed. Generating a second assembly is more of an escape hatch for pre-compilation but the generated descriptors need to match the ones that are created in the non-split assembly. Soooo :+1: